packages/preprocess/src/utils/get-route-hash.spec.cjs

```diff
@@ -0,0 +1,20 @@
+const md5 = require("blueimp-md5")
+const {faker} = require("@faker-js/faker")
+const {getRouteHash} = require("./get-route-hash.cjs")
+
+describe("get-route-hash", () => {
+    it("should hash /src/pages/+page.md to equal md5('/')", () => {
+        const result = getRouteHash("/src/pages/+page.md")
+        expect(result).toEqual(md5("/"))
+    })
+
+    it("should hash /src/pages/some-route/+page.md to equal md5('/some-route')", () => {
+        const result = getRouteHash("/src/pages/some-route/+page.md")
+        expect(result).toEqual(md5("/some-route"))
+    })
+
+    it("should hash /src/pages/random-string/+page.md the same way twice", () => {
+        const route = `/src/pages/${faker.company.bsAdjective()}-${faker.company.bsBuzz()}-${faker.company.bsNoun()}/+page.md`
+        expect(getRouteHash(route)).toEqual(getRouteHash(route))
```

Comment:
As `md5` should always be the same, it would be okay to have getRouteHash(route) and a hard-coded string
e.g. `5d41402abc4b2a76b9719d911017c592`

Reply:
This is more about ensuring that the route is processed exactly the same way twice, but adding a test case for a hardcoded string isn't a bad idea.